aerleon/lib/junipermsmpc.py

```diff
@@ -399,7 +402,7 @@ def _BuildTokens(self):
         )
         return supported_tokens, supported_sub_tokens
 
-    def _BuildPort(self, ports):
+    def _BuildPort(self, ports: List[Union[Any, List[int], Tuple[int, int]]]) -> List[str]:
```

Comment:
Monkeytype seems to have noticed this function is called with lists instead of tuples sometimes, and sure enough on line 616 (pre-changes), `_TranslatePolicy` constructs a list instead of a tuple to represent a port range. I'd suggest clamping the type to `List[Tuple[int, int]]` and then cleaning up line 616 to use a tuple.

Reply:
done